<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiScaleRetinexScale</class>
 <widget class="QWidget" name="MultiScaleRetinexScale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>81</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="indexLabel">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Gauss-Width</string>
     </property>
     <widget class="QDoubleSpinBox" name="sigmaSpinbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>72</width>
        <height>28</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>4.000000000000000</double>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Weight</string>
     </property>
     <widget class="QDoubleSpinBox" name="omegaSpinbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>72</width>
        <height>28</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.400000000000000</double>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
